DIKI_ALTGR is gone.
Now application developers don't need to check for both DIKI_ALTGR and
DIKI_ALT_R if they want to check for the key right next to the space bar :)
(Priyank)

--- ClanLib-0.6.5/Sources/Display/Input/DirectFB/keyboard_directfb.cpp	2006-05-06 17:20:38.000000000 +0530
+++ ClanLib-0.6.5/Sources/Display/Input/DirectFB/keyboard_directfb.cpp	2006-05-06 17:20:49.000000000 +0530
@@ -243,7 +243,6 @@
     case DIKI_SHIFT_R:     return CL_KEY_RSHIFT;
     case DIKI_ALT_L:       return CL_KEY_ALT;
     case DIKI_ALT_R:       return CL_KEY_ALT;
-    case DIKI_ALTGR:       return CL_KEY_ALTGR;
     case DIKI_TAB:         return CL_KEY_TAB;
     case DIKI_ENTER:       return CL_KEY_ENTER;
     case DIKI_SPACE:       return CL_KEY_SPACE;
@@ -337,7 +336,6 @@
     case CL_KEY_LSHIFT:    return DIKI_SHIFT_L;
     case CL_KEY_RSHIFT:    return DIKI_SHIFT_R;
     case CL_KEY_ALT:       return DIKI_ALT_L;
-    case CL_KEY_ALTGR:     return DIKI_ALTGR;
     case CL_KEY_TAB:       return DIKI_TAB;
     case CL_KEY_ENTER:     return DIKI_ENTER;
     case CL_KEY_SPACE:     return DIKI_SPACE;